docs/manifest/routine.md

```
@@ -754,3 +759,126 @@ Cron triggers are already declarative + versioned with the routine via
+      type: string
+      default: "Unify - Účetnictví"
+  slash:
+    enabled: true
+    label: Monthly accounting pack
+    label_cs: Účetní podklady za měsíc
+    icon: receipt
+```
+
+That routine is then offered as `/msn-etn-podklady` in both chat and the repl:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)